core/src/main/scala/org/apache/spark/util/ActorLogReceive.scala

```diff
@@ -43,7 +43,13 @@ private[spark] trait ActorLogReceive {
 
     private val _receiveWithLogging = receiveWithLogging
 
-    override def isDefinedAt(o: Any): Boolean = _receiveWithLogging.isDefinedAt(o)
+    override def isDefinedAt(o: Any): Boolean = {
+      val handled = _receiveWithLogging.isDefinedAt(o)
+      if (!handled) {
+        log.warn(s"Received unexpected actor system event: $o")
```

Comment:
@andrewor14 @srowen I think that down to info or debug is ok. What do you think? Can I change it? Wait for now?